qiskit/execute.py

```diff
@@ -31,7 +32,8 @@ def execute(experiments, backend,
             qobj_id=None, qobj_header=None, shots=1024,  # common run options
             memory=False, max_credits=10, seed_simulator=None,
             default_qubit_los=None, default_meas_los=None,  # schedule run options
-            schedule_los=None, meas_level=2, meas_return='avg',
+            schedule_los=None, meas_level=MeasLevel.CLASSIFIED.value,
```

Comment:
Maybe the default should be the enumerated type, then the implementation could take either the enumerated type or the value. In the later case it could issue a deprecation warning.

Reply:
I like the idea of accepting the type. I was inspired a bit by log levels and such, where a raw value is as acceptable as the param -- I guess in that case `MeasLevel.CLASSIFIED` would have to be equal to 2 to keep this as simple as possible. Like:
```
class MeasLevel:
    RAW = 0
    KERNELED = 1
    CLASSIFIED = 2
```
not even an Enum. What do you think?

Reply:
yeah please let's keep both Enum and integer working (the latter is on the Arxiv API definition). It should be natural with a mapping like above.